Piwik.Tracker/PiwikTracker.cs

```diff
@@ -182,7 +182,7 @@ public class PiwikTracker
         private bool _configCookiesDisabled;
         private string _configCookiePath = DefaultCookiePath;
         private string _configCookieDomain = "";
-        private readonly long _currentTs = (long)(DateTime.Now - new DateTime(1970, 1, 1)).TotalSeconds;
+        private readonly long _currentTs = (long)(DateTime.UtcNow - new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc)).TotalSeconds;

```

Comment:
I agree with your point. In that case, could we maybe factor out the construction of such a date in a common function to:
- make the code more readable by attributing this date value a name
- promote the usage of UTC